contracts/0.4.24/interfaces/ILidoOracle.sol

```diff
@@ -55,6 +55,11 @@ interface ILidoOracle {
         uint128 beaconBalance,
         uint128 beaconValidators
     );
+    event ReportPushed(
+         uint256 postTotalPooledEther,
+         uint256 preTotalPooledEther,
+         uint256 timeElapsed,
+         uint256 totalShares);
```

Comment:
how about `postTotalShares` for clarity?

Reply:
Sure, will rename it. Just a question: should the first three parameters still be reported (`postTotalPooledEther, preTotalPooledEther, timeElapsed`)?